### PR TITLE
Add Release .NET 6 label to breaking change issues

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: .repoman.yml [hunk 1]
```diff
@@ -43,6 +43,17 @@ issues:
       pass:
         - labels-add: ["source incompatible"]
 
+    # Checks breaking change version and adds a label
+    - check:
+        - type: query
+          value: "contains(Issue.body, '.NET 6 Preview 7') == `true`
```
Comment (adegeo) [inline .repoman.yml:49]: I believe YAML needs special syntax to make mutliline strings. I'm also not sure if JMESPath handles multiline.

But why not just use a simple:

```
contains(Issue.body, '.NET 6 Preview') == `true`
```

Comment (gewarren) [reply .repoman.yml:49]: We can probably just use the simple form. I'll do another commit.

**Review by tdykstra — APPROVED** [review]
LGTM

**Comment by gewarren** [comment]
Thanks Tom!

**Pull request merged**